PWGCF/DataModel/CorrelationsDerived.h

```diff
@@ -79,6 +79,39 @@ using CFTrack = CFTracks::iterator;
 using CFTrackLabel = CFTrackLabels::iterator;
 using CFTracksWithLabel = soa::Join<CFTracks, CFTrackLabels>;
 using CFTrackWithLabel = CFTracksWithLabel::iterator;
+
+//------transient CF-filter to CF-2prong-filter
+DECLARE_SOA_TABLE(CFCollRefs, "AOD", "CFCOLLREF", o2::soa::Index<>, track::CollisionId); //! Transient cf collision index table
+
+using CFCollRef = CFCollRefs::iterator;
+
+namespace cftrackref
+{
+DECLARE_SOA_INDEX_COLUMN(Track, track);
```

Comment:
This one we cannot reuse from the data model itself?

Reply:
There's `track::CollisionId` but what could be the corresponding `TrackId`? (trying to find one in "AO2D tables")

Reply:
You could use for example `o2::ambigous::TrackId` I think from AnalysisDataModel.h

Reply:
But if you find it more clean to redefine it, that is also OK for me

Reply:
I saw the `ambiguous` but the description for it in the documentation implied something else so didn't think it was the right choice.